src/app/join/hooks/useServiceAgreeCheck.ts

```
@@ -3,7 +3,7 @@ import { useEffect, useState } from 'react';
 import { ServiceAgreeCheck } from '../JoinPage.types';
 
 import { ROLE } from '@/constants/config';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)